src/OrleansProviders/Streams/Memory/MemoryMessageBody.cs

```diff
@@ -29,16 +30,28 @@ public interface IMemoryMessageBodySerializer
     /// Default IMemoryMessageBodySerializer
     /// </summary>
     [Serializable]
-    public class DefaultMemoryMessageBodySerializer : IMemoryMessageBodySerializer
+    public class DefaultMemoryMessageBodySerializer : IMemoryMessageBodySerializer, IOnDeserialized
```

Comment:
@jason-bragg I'm happy with that approach. I didn't want to change the interface, since it implies that it's supposed to be serializer agnostic.

I don't want to have to add `IOnDeserialized` just for this.